<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nodePopupClass</class>
 <widget class="QMainWindow" name="nodePopupClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="nodeTreeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>371</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="indentation">
     <number>20</number>
    </property>
    <property name="rootIsDecorated">
     <bool>true</bool>
    </property>
    <property name="uniformRowHeights">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>157</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>157</number>
    </attribute>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>IP Address</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>611</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>7</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>7</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QPushButton" name="defaultsButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>7</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore Defaults</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>7</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>15</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="profileEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>15</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="nodeGroupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Node Settings</string>
    </property>
    <widget class="QTreeWidget" name="profileTreeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>291</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Profiles</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="uptimeLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>57</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Uptime</string>
     </property>
    </widget>
    <widget class="QLabel" name="ethernetLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>57</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ethernet</string>
     </property>
    </widget>
    <widget class="QLabel" name="tcpActionLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Default TCP Action</string>
     </property>
    </widget>
    <widget class="QLabel" name="uptimeBehaviorLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Uptime Behavior</string>
     </property>
    </widget>
    <widget class="QLabel" name="ipLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QComboBox" name="uptimeBehaviorComboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Flat Value</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random in Range</string>
      </property>
     </item>
    </widget>
    <widget class="QTextEdit" name="uptimeEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="tcpActionEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="ethernetEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="ipEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTreeWidget" name="portTreeWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>65</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>65</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Port</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Behavior</string>
      </property>
     </column>
    </widget>
    <widget class="QTextEdit" name="personalityEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="personalityLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Personality</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cloneButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>378</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editPortsButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>378</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Ports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>378</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>378</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>378</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>378</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
